app/dashboard/static/js/app/view-tests-board-job-kernel.2017.7.2.js

```diff
@@ -0,0 +1,534 @@
+            aNode.appendChild(html.boot());
+            tooltipNode.appendChild(aNode);
+
+            domNode.appendChild(tooltipNode);
+            domNode.insertAdjacentHTML('beforeend', '&nbsp;&dash;&nbsp;');
+
+            tooltipNode = html.tooltip();
+            tooltipNode.setAttribute(
+                'title',
+                'Build reports for &#171;' + gJob + '&#187; - ' + gKernel);
+            aNode = document.createElement('a');
+            aNode.setAttribute(
+                'href', '/build/' + gJob + '/kernel/' + gKernel);
+            aNode.appendChild(html.build());
+            tooltipNode.appendChild(aNode);
+
+            domNode.appendChild(tooltipNode);
+
+            html.replaceContent(
+                document.getElementById('git-describe'), domNode);
+
+            // Git URL.
+            if (gitURLs[0]) {
+                aNode = document.createElement('a');
+                aNode.setAttribute('href', gitURLs[0]);
+                aNode.appendChild(document.createTextNode(gitURL));
+                aNode.insertAdjacentHTML('beforeend', '&nbsp;');
+                aNode.appendChild(html.external());
+            } else {
+                if (gitURL && gitURL !== undefined) {
+                    aNode = document.createTextNode(gitURL);
+                } else {
+                    aNode = html.nonavail();
+                }
+            }
+            html.replaceContent(document.getElementById('git-url'), aNode);
+
+            // Git commit.
+            if (gitURLs[1]) {
+                aNode = document.createElement('a');
+                aNode.setAttribute('href', gitURLs[1]);
+                aNode.appendChild(document.createTextNode(gitCommit));
+                aNode.insertAdjacentHTML('beforeend', '&nbsp;');
+                aNode.appendChild(html.external());
+            } else {
+                if (gitCommit && gitCommit !== null) {
+                    aNode = document.createTextNode(gitCommit);
+                } else {
+                    aNode = html.nonavail();
+                }
+            }
+            html.replaceContent(document.getElementById('git-commit'), aNode);
+
+            // Date.
+            domNode = document.createElement('time');
+            domNode.setAttribute('datetime', createdOn.toISOString());
+            domNode.appendChild(
+                document.createTextNode(createdOn.toCustomISODate()));
+            html.replaceContent(
+                document.getElementById('job-date'), domNode);
+        }
+    }
+
+    function getTests() {
+        var data;
+        var deferred;
+
+        function getData(lab) {
+            data.lab_name = lab;
+
+		    gLabTable.push(table({
+		        tableId: 'lab-table-' + lab,
+		        tableLoadingDivId: 'lab-table-loading-' + lab,
+		        tableDivId: 'lab-table-div-' + lab
+		    }));
+		    var labTable = gLabTable[gLabTable.length - 1];
```

Comment:
Better move the `var` declaration  outside of this function definition, where the other two variables are define, or at least at the beginning of this function.